commit 6a44473384e8959fa3d67903653e77d81d3e7019
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2019-08-29T22:32:55+00:00 (6 weeks ago)
    
    joyent/bugview#37 illumos-live links should be rewritten to point at smartos-live

diff --git a/jirapub.js b/jirapub.js
index 740e206..9a8adee 100644
--- a/jirapub.js
+++ b/jirapub.js
@@ -569,6 +569,11 @@ fix_url(input)
 				m: '/smartos-live',
 				p: '/joyent/smartos-live'
 			},
+			{
+				h: 'github.com',
+				m: '/illumos-live',
+				p: '/joyent/smartos-live'
+			},
 			{
 				h: 'github.com',
 				m: '/illumos-extra',
